cranelift/codegen/src/verifier/mod.rs

```diff
@@ -1536,28 +1535,6 @@ impl<'a> Verifier<'a> {
                             ));
                         }
                     }
-                    Opcode::Breduce | Opcode::Ireduce | Opcode::Fdemote => {
```

Comment:
I don't think we want to delete the `Ireduce` and `Fdemote` cases here?

Reply:
Good catch, I got a bit overzealous there :)